themes/sdss_subtheme/src/scss/components/header/_header.scss

```diff
@@ -0,0 +1,299 @@
+        background: $su-color-white;
+      }
+    }
+
+    section {
+      &::after {
+        left: 100%;
+        background: $sdss-color-blue-light;
+      }
+    }
+
+  }
+}
+
+// SDSS other sites header
+@include grid-media-min('lg') {
+  .fixed-header.su-lockup--option-a,
+  .fixed-header.su-lockup--option-b {
+    margin: 0 auto;
+    line-height: 2em;
+    padding: 0;
+
+    // logo and taglines
+    .su-lockup__cell1,
+    .su-lockup__cell2 {
+      border-right: 0;
+      border-left: 0;
+    }
+
+    .su-lockup__custom-logo {
+      max-width: 166px;
+    }
+
+    .su-lockup__cell1 {
+      align-self: center;
+      margin-right: 30px;
+      max-width: 192px;
+    }
+
+    .su-lockup__cell2 {
+      margin-right: 30px;
+    }
+
+    .su-lockup__line1,
+    .su-lockup__line2 {
+      color: $su-color-white;
+    }
+
+    .su-lockup__line5 {
+      display: none;
+    }
+
+    .su-lockup__cell2 {
+      border-left: 0;
+      background-color: $sdss-color-palo-alto-dark;
+      padding: 30px 20px 20px;
+      max-width: unset;
+  }
+
+    .sdss-masthead {
+      display: grid;
+      grid-template-columns: 1fr;
+      grid-template-areas:
+      "lockup lockup lockup"
+      "main-nav main-nav search";
+      align-items: flex-end;
+      column-gap: 40px;
+
+      .su-lockup {
+        grid-area: lockup;
+
+        background: $su-color-white;
+        width: unset;
+        max-width: unset;
+        padding: 0; //spacing to get the background-color.
+        margin-right: -20px;
+      }
+
+      #sdss-button--search-link {
+        grid-area: search;
+        width: 32px;
+        height: 32px;
+        position: relative;
+        top: 0;
+        right: unset;
+        align-self: center;
+        background-color: $sdss-color-palo-alto-dark;
+        border-radius: 32px;
+      }
+
+      .su-multi-menu {
+        grid-area: main-nav;
+      }
+    }
+
+    nav+.su-site-search, ul li .su-site-search {
+      display: none;
+    }
+
+    // Header color outside in the margins.
+    .su-lockup__cell2 {
+      position: relative; /* child absolute */
+    }
+
+    .su-lockup__cell2 {
+      &::after{
+        content: "";
+        position: absolute;
+        /* fill vertically */
+        top: 0;
+        bottom: 0;
+        width: 9600px;
+        right: 100%;
+        background: $sdss-color-palo-alto-dark;
+      }
+    }
+
+    .su-lockup__cell2 {
+      &::after {
+        left: 100%;
+        background: $sdss-color-palo-alto-dark;
+      }
+    }
+  }
+}
+
+// SDSS other sites header responsive tweak.
+@include grid-media-max('lg') {
+  .fixed-header.su-lockup--option-a,
+  .fixed-header.su-lockup--option-b {
+    .su-masthead .su-lockup--option-a .su-lockup__cell2 {
+      border-left: none;
+      padding-bottom: 1em;
+      padding-left: 1em;
+    }
+  }
+}
+
+// Removed line 5 from all displays
+.fixed-header.su-lockup--option-a,
+.fixed-header.su-lockup--option-b {
+  .su-lockup__line5 {
+    display: none;
+  }
+}
```

Comment:
Do you think there's another way to do this without hiding with CSS?

Reply:
1. Link: I believe that is the same functionality as we currently have. I will look at the request again to see if anything is called out. 
2. I showed Opt B and how it looked to Design and they thought it was fine since it won't be used right away. There were no designs for this piece. 
3. For the line5 that is done all over in the lockup.  

I'm going to take a look and see what I can do.